ios/AppLovinMAX.m

```diff
@@ -751,12 +751,21 @@ - (instancetype)init
         [self logUninitializedAccessError: @"setBannerExtraParameter"];
         return;
     }
-
+    
     [self setAdViewExtraParameterForAdUnitIdentifier: adUnitIdentifier adFormat: DEVICE_SPECIFIC_ADVIEW_AD_FORMAT key: key value: value];
 }
 
-RCT_EXPORT_METHOD(setBannerLocalExtraParameter:(NSString *)adUnitIdentifier :(NSString *)key :(nullable NSString *)value)
+RCT_EXPORT_METHOD(setBannerLocalExtraParameter:(NSString *)adUnitIdentifier :(NSDictionary<NSString *, id> *)object)
 {
+    if ( !self.sdk )
+    {
+        [self logUninitializedAccessError: @"setBannerLocalExtraParameter"];
+        return;
+    }
+    
+    NSString *key = [[object allKeys] objectAtIndex: 0];
+    id value = [object objectForKey:key];
```

Comment:
You can use `parameterDict.allValues.firstObject`.

Reply:
Updated with:

```
    NSString *key = parameterDict.allKeys.firstObject;
    id value = parameterDict.allValues.firstObject;
```